src/table.js

```diff
@@ -19,13 +19,20 @@ function Table(props) {
         props.onClickIcon(e.target.parentElement.parentElement.id, props.id);
     }
 
-    const handleRowClick = (e) => {
+    const handleChevronClick = (e) => {
         e.preventDefault();
                 
         // pass in row index and table index
         props.onClickIcon(e.target.parentElement.parentElement.id, props.id);
     }
 
+    // const handleRowClick = (e) => {
```

Comment:
What was the bug with this one? Was it a weird click propagation issue?

Reply:
Yeah, if the click was on the row text instead of just the table cell, the function wasn't running properly. Should be able to fix this one soon!

Reply:
sweet ~ nothing some stopEventPropagation can't solve, hopefully ✨